### PR TITLE
Added a uuid to the session ID to prevent collisions from users at th…

### PR DESCRIPTION
**Pull request by jworcestBAH**
…e same IP

### CODE REVIEW

**Comment by cmc333333** [comment]
Would it make sense to drop the `_create_identifier()` call and just use a UUID for the session id?

File: dataactbroker/handlers/aws/session.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def save_session(self, app, session, response):
             else:
                 expiration = datetime.utcnow() + timedelta(seconds=SessionTable.TIME_OUT_LIMIT)
         if(not "_uid" in session):
-            session["_uid"] = _create_identifier()
+            session["_uid"] = "{}|{}".format(_create_identifier(),uuid4())
```
Comment (cmc333333) [inline dataactbroker/handlers/aws/session.py]: Small detail, but rather than duplicating the logic, could we call `self.resetId(session)`?

**Comment by jworcestBAH** [comment]
I thought about that, but we have a check in 'isSessionSecure' that checks whether the _create_identifier() part matches their session id to prevent session ids from being intercepted and used to mimic a user (this includes user's IP).  To keep that functionality available I combined the two.

File: dataactbroker/handlers/aws/session.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactbroker/handlers/aws/session.py]: Good call, changed to call this

**Comment by cmc333333** [comment]
Gotcha; agreed that that's outside the scope for this PR, but I created DB-1287 to track it for later. This is good as is, though.

**Pull request merged**